features/logging/elasticsearch.feature

```diff
@@ -44,7 +44,10 @@ Feature: Elasticsearch related tests
     And the expression should be true> cluster_logging('instance').logstore_storage_class_name == cb.default_sc.name
     Given I wait for the "elasticsearch" elasticsearches to appear
     And the expression should be true> elasticsearch('elasticsearch').nodes[0]['storage']['storageClassName'] == cb.default_sc.name
-    Given I wait for clusterlogging with "fluentd" log collector to be functional in the project
+    #Given I wait for clusterlogging with "fluentd" log collector to be functional in the project
```

Comment:
EO is unable to update the Kibana status(https://bugzilla.redhat.com/show_bug.cgi?id=1849564), so I use the below 3 steps to replace the step `I wait for clusterlogging with "fluentd" log collector to be functional in the project`.

Reply:
@QiaolingTang once the bug is fixed then we can use the `old` approach, right?  If it works, then I don't recommend removing the various functions.

Reply:
Yes, this step should work once the bug is fixed,  but I'm not removing the functions related to this step, I just want to remove some useless functions, we never used them.